pit30m/data/log_reader.py

```diff
@@ -242,13 +243,14 @@ def continuous_pose_dense(self) -> np.ndarray:
 
         Not useful for global localization, since each log will have its own coordinate system, but useful for SLAM-like
         evaluation, since you will have a continuous pose trajectory.
+
+        Pose times are UNIX seconds.
         """
         pose_data = []
-        # TODO(andrei): Document the timestamps carefully.
         for pose in self.raw_pose_data:
             pose_data.append(
                 (
-                    pose["capture_time"],
+                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```

Comment:
```suggestion
                    # TODO(julieta) The overhead of this conversion might be very large at scale. Consider vectorizing
                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```

Reply:
Or maybe we could leave everything on GPS timestamps? Aren't they standard in robotics?

Reply:
eh, I think they sort of are but I think they may be confusing to vision/ML people